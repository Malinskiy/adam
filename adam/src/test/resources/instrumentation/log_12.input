INSTRUMENTATION_STATUS: class=com.example.ClassIgnoredTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: numtests=1
INSTRUMENTATION_STATUS: stream=
com.example.ClassIgnoredTest:
INSTRUMENTATION_STATUS: test=null
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: class=com.example.ClassIgnoredTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: numtests=1
INSTRUMENTATION_STATUS: stream=
com.example.ClassIgnoredTest:
INSTRUMENTATION_STATUS: test=null
INSTRUMENTATION_STATUS_CODE: -3
INSTRUMENTATION_RESULT: stream=

Time: 0.001

OK (0 tests)


Error: Failed to generate Emma/JaCoCo coverage. 
INSTRUMENTATION_CODE: -1
