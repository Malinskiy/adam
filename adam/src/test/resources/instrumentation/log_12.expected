TestRunStartedEvent(testCount=1)
TestStarted(id=TestIdentifier(className=com.example.ClassIgnoredTest, testName=null))
TestIgnored(id=TestIdentifier(className=com.example.ClassIgnoredTest, testName=null))
TestEnded(id=TestIdentifier(className=com.example.ClassIgnoredTest, testName=null), metrics={})
TestRunEnded(elapsedTimeMillis=1, metrics={})
